README.md

```diff
@@ -22,10 +22,11 @@ Following is a list of Greengrass components.
 * [Kinesis Video Streams](https://github.com/awslabs/aws-greengrass-labs-kvs-stream-uploader): This component ingests audio and video streams from RTSP cameras connected to the Greengrass core device. The audio and video streams are then uploaded to Amazon Kinesis Video Streams.
 * [Modbus TCP](https://github.com/awslabs/aws-greengrass-labs-modbus-tcp-protocol-adapter): This component collects data from local devices through the ModbusTCP protocol and publishes it to selected data streams.
 * [Bluetooth IoT Gateway](https://github.com/awslabs/aws-greengrass-labs-bluetooth-gateway): The BLE IoT Gateway component uses the [BluePy](https://ianharvey.github.io/bluepy-doc/index.html) library (Python module which allows communication with Bluetooth Low Energy devices) to create the BLE client interface.
-* [Grafana](https://github.com/awslabs/aws-greengrass-labs-dashboard-grafana): This component provides support for the Grafana visualization dashboard at the edge. Customers can use this component to visualize data locally. This component can be used to render data from the InfluxDB component.
 * [Home Assistant](https://github.com/awslabs/aws-greengrass-labs-component-for-home-assistant): This component enables use cases where the customer desires [Home Assistant](https://www.home-assistant.io/) for local control of their smart home devices. This component enables deep integration with AWS services at the edge and in the cloud, to deliver powerful home automation solutions that extend Home Assistant’s capabilities.
+* [InfluxDBGrafana Dashboard](https://github.com/awslabs/aws-greengrass-labs-dashboard-influxdb-grafana): This component provides a one-click experience for the below InfluxDB and Grafana components, connecting InfluxDB to Grafana and automating the setup of a local Grafana dashboard rendering Greengrass telemetry in real-time.
```

Comment:
How about adding hyperlink to InfluxDB and Grafana components ?

Reply:
done